docs/compliance/nist/itl_october2017.md

```diff
@@ -0,0 +1,7 @@
+---
+description: NIST ITL October 2017 NIST Guidance on Application Container Security
+keywords: standards, compliance, security
+title: NIST ITL October 2017 guidance on application container security
+---
+
+The NIST [Information Technology Laboratory (ITL) Bulletin dated October 2017](https://csrc.nist.gov/publications/detail/itl-bulletin/2017/10/application-container-security/final), NIST Guidance on Application Container Security, is a summarization of the content made available in [NIST SP 800-190](800_190.md) and [NISTIR 8176](nistir_8176.md). It provides a fundamental, high-level starting point from which to evaluate the primary security enhancements that containers provide.
```

Comment:
I think publication titles should be in italics.

Reply:
Updating